TextToSpeech/TextToSpeech.cpp

```diff
@@ -86,37 +81,24 @@ namespace WPEFramework
             registerMethod("setTTSConfiguration", &TextToSpeech::setTTSConfiguration, this);
             registerMethod("getTTSConfiguration", &TextToSpeech::getTTSConfiguration, this);
             registerMethod("isTTSEnabled", &TextToSpeech::isTTSEnabled, this);
-            registerMethod("isSessionActiveForApp", &TextToSpeech::isSessionActiveForApp, this);
-            registerMethod("acquireResource", &TextToSpeech::acquireResource, this);
-            registerMethod("claimResource", &TextToSpeech::claimResource, this);
-            registerMethod("releaseResource", &TextToSpeech::releaseResource, this);
-            registerMethod("createSession", &TextToSpeech::createSession, this);
-            registerMethod("destroySession", &TextToSpeech::destroySession, this);
-            registerMethod("isActiveSession", &TextToSpeech::isActiveSession, this);
-            registerMethod("setPreemptiveSpeak", &TextToSpeech::setPreemptiveSpeak, this);
             registerMethod("speak", &TextToSpeech::speak, this);
+            registerMethod("cancel", &TextToSpeech::cancel, this);
             registerMethod("pause", &TextToSpeech::pause, this);
             registerMethod("resume", &TextToSpeech::resume, this);
-            registerMethod("abort", &TextToSpeech::abort, this);
             registerMethod("isSpeaking", &TextToSpeech::isSpeaking, this); 
             registerMethod("getSpeechState", &TextToSpeech::getSpeechState, this);
+            registerMethod("setPreemptiveSpeak", &TextToSpeech::setPreemptiveSpeak, this);
             registerMethod("requestExtendedEvents", &TextToSpeech::requestExtendedEvents, this);
         
-            if(!m_sessionCallback)
-                m_sessionCallback = new TTSSessionServiceCallback(this);
-
             if(!m_connectionCallback)
                 m_connectionCallback = new TTSConnectionCallback(this);
 
             if(!m_ttsManager) {
                 m_ttsManager = TTS::TTSManager::create(m_connectionCallback);
-                if(m_ttsManager)
-                {
-                    m_policy = getResourceAllocationPolicy();
-                    LOGINFO("%d(%s) policy is enforced by TTSEngine", m_policy, policyStr(m_policy));
-                    if(m_connectionCallback)
-                        m_connectionCallback->onTTSServerConnected();   
-                }
+
+                //FIXME Hardcoding of this enableTTS will be removed once Resident application is implemented.
+                m_ttsEnabled = true;
```

Comment:
Probably it'd be better to have it disabled by default and 'set' curl call to enable it for testing.